app/controllers/items_controller.rb

```diff
@@ -68,4 +73,11 @@ def item_params
     filter[:section_id] = nil if filter[:section_id] == ''
     filter
   end
+
+  def check_authorization
+    return if current_user.admin
+    return if @item.medium && current_user.can_edit?(@item.medium)
+    return if @item.medium.nil?
+    redirect_to :root, alert: I18n.t('controllers.unauthorized')
```

Comment:
why is this authorisation check carried out in this fashion and not by using CanCanCan

Reply:
Indeed. It has been moved there.